src/main/js/controls/Bar/Bar.js

```diff
@@ -191,7 +193,8 @@ class Bar extends GraphContent {
             this.ordinalScale,
             graph.svg,
             this.dataTarget,
-            graph.config
+            graph.config,
+            settingsDictionary(graph.config).transition
```

Comment:
This needs to be a 1 time call in GraphConfig since you wont expect any different output no matter where you call. You can store the output as part of the config [here](https://github.com/cerner/carbon-graphs/blob/a47fc48079b8970f8f5e538ff402cb2b79107b45/src/main/js/controls/Graph/GraphConfig.js#L85). 
```javascript
config.settingsDictionary = settingsDictionary(input);
```
Now you will be able to directly reference `graph.config.settingsDictionary.shouldClamp` or `graph.config.settingsDictionary.transition` wherever you need (in graph or any content).

Reply:
I will make the changes